region/lowernorfair/east.json

```diff
@@ -1656,11 +1656,11 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  "Super",
                   {"cost": [
-                    {"heatFrames": 1200}
+                    {"heatFrames": 400}
```

Comment:
Additionally, should this be a tech?

Reply:
My suggestion is canHitBox here and canPlasmaHitBox at Kihunters.